nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -29,11 +29,11 @@ The Node Maintenance Operator is supported in a restricted network environment.
 
 [id="setting-node-in-maintenance-mode"]
 == Setting a node to maintenance mode
-You can place a node into maintenance from the CLI or by using a `NodeMaintenance` CR.
+You can place a node into maintenance from the web console or in the CLI by using a `NodeMaintenance` CR.
 
-include::modules/eco-setting-node-maintenance-cli.adoc[leveloffset=+2]
+include::modules/eco-setting-node-maintenance-cr-web-console.adoc[leveloffset=+2]
 
-include::modules/eco-setting-node-to-maintenance-mode-with-cr.adoc[leveloffset=+2]
+include::modules/eco-setting-node-maintenance-cr-cli.adoc[leveloffset=+2]
 
 You can check the status of current `NodeMaintenance` CR tasks.
```

Comment:
Can you check this? I don't think you want it showing up at the end of the module. It shows up as the intro sentence in the module that follows. 
![image](https://user-images.githubusercontent.com/47002677/156682227-7d7a0a4c-130f-401b-aead-719c2e1b3c79.png)